INFO:     Will watch for changes in these directories: ['/mnt/e/work/hackathons/hyperverge-sensai/sensai-ai']
INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
INFO:     Started reloader process [161900] using StatReload
Process SpawnProcess-1:
Traceback (most recent call last):
  File "/home/harsh/miniconda3/lib/python3.12/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/home/harsh/miniconda3/lib/python3.12/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/mnt/e/work/hackathons/hyperverge-sensai/sensai-ai/venv/lib/python3.12/site-packages/uvicorn/_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "/mnt/e/work/hackathons/hyperverge-sensai/sensai-ai/venv/lib/python3.12/site-packages/uvicorn/server.py", line 67, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/harsh/miniconda3/lib/python3.12/asyncio/runners.py", line 195, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/home/harsh/miniconda3/lib/python3.12/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/harsh/miniconda3/lib/python3.12/asyncio/base_events.py", line 691, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/mnt/e/work/hackathons/hyperverge-sensai/sensai-ai/venv/lib/python3.12/site-packages/uvicorn/server.py", line 71, in serve
    await self._serve(sockets)
  File "/mnt/e/work/hackathons/hyperverge-sensai/sensai-ai/venv/lib/python3.12/site-packages/uvicorn/server.py", line 78, in _serve
    config.load()
  File "/mnt/e/work/hackathons/hyperverge-sensai/sensai-ai/venv/lib/python3.12/site-packages/uvicorn/config.py", line 436, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/e/work/hackathons/hyperverge-sensai/sensai-ai/venv/lib/python3.12/site-packages/uvicorn/importer.py", line 22, in import_from_string
    raise exc from None
  File "/mnt/e/work/hackathons/hyperverge-sensai/sensai-ai/venv/lib/python3.12/site-packages/uvicorn/importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/harsh/miniconda3/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/mnt/e/work/hackathons/hyperverge-sensai/sensai-ai/src/api/main.py", line 10, in <module>
    from api.routes import (
  File "/mnt/e/work/hackathons/hyperverge-sensai/sensai-ai/src/api/routes/auth.py", line 3, in <module>
    from api.db.user import insert_or_return_user
  File "/mnt/e/work/hackathons/hyperverge-sensai/sensai-ai/src/api/db/user.py", line 17, in <module>
    from api.slack import send_slack_notification_for_new_user
  File "/mnt/e/work/hackathons/hyperverge-sensai/sensai-ai/src/api/slack.py", line 3, in <module>
    from api.settings import settings
  File "/mnt/e/work/hackathons/hyperverge-sensai/sensai-ai/src/api/settings.py", line 3, in <module>
    from pydantic_settings import BaseSettings, SettingsConfigDict
ModuleNotFoundError: No module named 'pydantic_settings'
INFO:     Stopping reloader process [161900]
